docs/src/tutorials/symmetric_mesh.md

```diff
@@ -0,0 +1,360 @@
+
+ Mesh Quality:
+         Measure         Minimum         Maximum         Average  Acceptable Low Acceptable High       Reference
+     Signed Area      0.00227157      0.06552808      0.01366911      0.00000000    999.99900000      1.00000000
+    Aspect Ratio      1.00389317      2.07480912      1.22443835      1.00000000    999.99900000      1.00000000
+       Condition      1.00048947      1.93007666      1.12853177      1.00000000      4.00000000      1.00000000
+      Edge Ratio      1.00607156      3.56541719      1.44892631      1.00000000      4.00000000      1.00000000
+        Jacobian      0.00158627      0.06202757      0.01120120      0.00000000    999.99900000      1.00000000
+   Minimum Angle     48.35050023     89.33703938     75.39251039     40.00000000     90.00000000     90.00000000
+   Maximum Angle     90.94150752    129.91539960    106.20679399     90.00000000    135.00000000     90.00000000
+       Area Sign      1.00000000      1.00000000      1.00000000      1.00000000      1.00000000      1.00000000
+```
+Note that the call to `generate_mesh` prints mesh quality statistics to the screen
+and updates the visualization.
+The background grid is *removed* from the visualization when the mesh is generated.
+
+!!! note "Mesh visualization"
+    Currently, only the "skeleton" of the mesh is visualized. Thus, the high-order curved boundary information
+    is not seen in the plot but this information **is present** in the mesh file generated.
+
+![first_reflect_mesh](https://github.com/trixi-framework/HOHQMesh.jl/assets/25242486/1afd159e-e8c4-459f-ba7d-a31057b0c135)
+
+!!! tip "Boundary names in the mesh file"
+    The boundary names of the original outer curves will be those defined by the user
+    in their construction above. The boundary labeled `"symmetry"` is now internal and
+    is marked appropriately as such. The reflected boundary names are appended
+    with `_R` (for reflected) in the mesh file. For instance, the reflected version
+    of the boundary `bottom` has the name `bottom_R` or the boundary named `circle` has the
+    reflected boundary counterpart named `circle_R`. These can be changed as desired by editing the mesh file.
+
+## Changing the reflection line
+
+It is also possible to create a symmetry boundary composed of multiple be co-linear segments.
+
+To change the line along which the mesh is reflected, we remove the current mesh
+that was just generated and re-plot the model curves
+and background grid.
+```julia
+remove_mesh!(symmetric_mesh)
+updatePlot!(symmetric_mesh, MODEL+GRID)
+```
+The `remove_mesh!` command deletes the mesh information from
+the interactive mesh project `symmetric_mesh` as well as the mesh file `symmetric_mesh.mesh`,
+control file `symmetric_mesh.control`, plot file `symmetric_mesh.tec`,
+and mesh statistics file `symmetric_mesh.txt` from the `out` folder.
+
+To illustrate the reflection about multiple boundary curves (which must be co-linear!),
```

Comment:
Is there a check for co-linearity? If not, it might be good to note that this is something that is not enforced and that the user is responsible for correct use.

If there is a check (which would make sense imho), it would be nice to inform the user about it as well here.

Reply:
Yes, an exception is thrown and the user is informed that the reflection was not performed. The user is also informed if the curve along which the reflection is requested is not a straight line. The tests for the latter are not exhaustive. It basically checks to see if the second derivative (that code already existed) is zero at a couple of points.